config/compilerFlags.cmake

```diff
@@ -72,6 +75,18 @@ if ( MINGW OR UNIX ) # MINGW, Linux, APPLE, CYGWIN
                     #" -Wconversion"
                     #" -Wold-style-cast"
                 )
+                # -Wdouble-promotion flag is not available in clang 3.4.2

```

Comment:
[question] Just out of curiosity. Which distribution is using such old version of clang ? :astonished: